### PR TITLE
[#6603] home/blocks: add new video block with template

### PR DESCRIPTION
**Pull request by phillimorland**
docs: https://docs.wagtail.org/en/stable/advanced_topics/embeds.html

### CODE REVIEW

File: digitalstrategie/settings/base.py [hunk 1]
```diff
@@ -173,6 +174,12 @@
 # Wagtail settings
 WAGTAIL_SITE_NAME = "digitalstrategie"
 WAGTAILIMAGES_IMAGE_MODEL = 'apps_images.CustomImage'
+WAGTAILEMBEDS_FINDERS = [
+    {
+        'class': 'wagtail.embeds.finders.oembed',
+        'providers': [youtube, vimeo],
+    }
+]
```
Comment (phillimorland) [inline digitalstrategie/settings/base.py:182]: not sure if this is the best place to add this?!

File: apps/home/templates/apps_home/blocks/video_block.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                  loading="lazy" />
+                <p class="overlay_text">
+                  {% if 'youtube' in value.video.url %}
+                    {% trans 'By viewing the video, you agree that your data will be transmitted to <strong>YouTube</strong>. For more information see' %}
+                    <a target="_blank" href="https://www.google.de/intl/de/policies/privacy/">{% trans 'Data protection' %}</a>.
+                  {% elif 'vimeo' in value.video.url %}
+                    {% trans 'By viewing the video, you agree that your data will be transmitted to <strong>Vimeo</strong>.For more information see' %}
+                    <a target="_blank" href="https://vimeo.com/privacy">{% trans 'Data protection' %}</a>.
+                  {% else %}
+                    {% trans 'DSGVO compliant videos are not supported by your browser.' %}
```
Comment (phillimorland) [inline apps/home/templates/apps_home/blocks/video_block.html:24]: it's probably more accurate to say 'This video cannot be viewed in a dsgvo compliant way, please click the video link if you wish to view? and add a link?

Comment (phillimorland) [reply apps/home/templates/apps_home/blocks/video_block.html:24]: swill check wording with Max after their standup

Comment (phillimorland) [reply apps/home/templates/apps_home/blocks/video_block.html:24]: it's an edge case as the embed does not allow non youtube/vimeo to be loaded but yes that is better, but was thinking a link would still be good, but will check with max, also I guess the formfield should have helptext too thinking about it

File: apps/home/templates/apps_home/blocks/video_block.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                  loading="lazy" />
+                <p class="overlay_text">
+                  {% if 'youtube' in value.video.url %}
+                    {% trans 'By viewing the video, you agree that your data will be transmitted to <strong>YouTube</strong>. For more information see' %}
+                    <a target="_blank" href="https://www.google.de/intl/de/policies/privacy/">{% trans 'Data protection' %}</a>.
+                  {% elif 'vimeo' in value.video.url %}
+                    {% trans 'By viewing the video, you agree that your data will be transmitted to <strong>Vimeo</strong>.For more information see' %}
+                    <a target="_blank" href="https://vimeo.com/privacy">{% trans 'Data protection' %}</a>.
+                  {% else %}
+                    {% trans 'DSGVO compliant videos are not supproted by your browser' %}
```
Comment (fuzzylogic2000) [inline apps/home/templates/apps_home/blocks/video_block.html]: There is a typo in supported

File: apps/home/templates/apps_home/blocks/video_block.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                  onclick="window.showVideo()">
+                    {% trans 'Accept and play' %}
+                </button>
+
+                <template>{% embed value.video.url "500" %}</template>
+            </div>
+            <div class="block-video__video" id="video-player{{ identifier }}"></div>
+        </div>
+        <p class="player-formats" id="player__{{ identifier }}">
+          {% trans 'Format:' %}
```
Comment (fuzzylogic2000) [inline apps/home/templates/apps_home/blocks/video_block.html:40]: Nice!

File: digitalstrategie/settings/base.py (hunk 1 above)
Comment (fuzzylogic2000) [reply digitalstrategie/settings/base.py:183]: Yes, I think it does fit with the images

**Review by fuzzylogic2000 — APPROVED** [review]
Works nicely and looks great!

File: locale/en/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -354,3 +386,32 @@ msgstr "Your search for"
 #: digitalstrategie/templates/search_results.html:70
 msgid "No results found"
 msgstr "No results found"
+
+#~ msgid "For more information see"
+#~ msgstr "For more information see"
+
+#~ msgid "To protect your privacy this video can\\\\"
+#~ msgstr "To protect your privacy this video can\\\\"
```
Comment (fuzzylogic2000) [inline locale/en/LC_MESSAGES/django.po:394]: The unused strings should be deleted from the files again.

**Review by fuzzylogic2000 — APPROVED** [review]
Cool! Still looks good and works great. :)
We can merge now and delete the strings later or you delete them and then merge, whatever you prefer!

**Review by goapunk — APPROVED** [review]
very cool!

**Pull request merged**